lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -1019,26 +1028,71 @@ spec = do
         expectResponseCode @IO HTTP.status404 r
         expectErrorMessage (errMsg404NoWallet $ w ^. walletId) r
 
+    it "TRANS_LIST_01x - Can list Incoming and Outgoing transactions" $ \ctx -> do
+        wSrc <- fixtureWallet ctx
+
+        r <- request @([ApiTransaction t]) ctx (listTxEp wSrc mempty)
+            Default Empty
+
+        -- let outs = getFromResponseList @(ApiTransaction t) 0 outputs r
+        -- let ins = getFromResponseList 0 inputs r
+        -- length outs `shouldBe` 10
+        -- length ins `shouldBe` 10
```

Comment:
... I'd like to assert for `inputs` and `outputs` size... But it doesn't type-check. I think the `input` and `output` lenses are wrong (getFromResponseList works fine on other lenses)